src/components/Checkbox/Checkbox.jsx

```diff
@@ -0,0 +1,67 @@
+import React, {Component, PropTypes} from 'react';
+import {PURE} from 'dx-util/src/react/pure';
+import {themr} from 'react-css-themr';
+import classnames from 'classnames';
+import {randomId} from 'dx-util/src/string/string';
+
+import Icon from '../Icon/Icon.jsx';
+
+export const CHECKBOX = Symbol('Checkbox');
+
+@PURE
+@themr(CHECKBOX)
+export default class Checkbox extends Component {
+	static propTypes = {
+		id: PropTypes.string,
+		checkboxIconName: PropTypes.string,
+		isDisabled: PropTypes.bool,
+		isChecked: PropTypes.bool,
+		onChange: PropTypes.func,
+		theme: PropTypes.shape({
+			container: PropTypes.string,
+			container_isDisabled: PropTypes.string,
+			input: PropTypes.string,
+			label: PropTypes.string,
+			view: PropTypes.string,
+			view_isDisabled: PropTypes.string,
+			checkboxIcon: PropTypes.string,
+			checkboxIcon_isChecked: PropTypes.string,
+			checkboxIcon_isDisabled: PropTypes.string
+		})
+	};
+
+	render() {
+		const {
+				id,
```

Comment:
make sure your settings **are the same** 
![image](https://cloud.githubusercontent.com/assets/1743568/18963984/b1bd86a8-867f-11e6-89d1-7f316477e995.png)